src/prpy/planning/retimer.py

```diff
@@ -154,6 +154,15 @@ def __init__(self, do_blend=True, do_shortcut=True, blend_radius=0.5,
             'time_limit': float(timelimit),
         })
 
+    @PlanningMethod
+    def RetimeTrajectory(self, robot, path, options=None, **kw_args):
+        from copy import deepcopy
+        if options is None:
+            options = {}
+        new_options = deepcopy(options)
+        new_options['time_limit'] = kw_args.get('timelimit', 3.)
+        return super(HauserParabolicSmoother, self).RetimeTrajectory(
+            robot, path, options=new_options, **kw_args)
```

Comment:
I agree that we should make it consistent (one way or the other). However, we've been living with the name mismatch for months and the change has nothing to do with this pull request. Let's keep this pull request as is and change the name in a separate PR.